Makefile

```diff
@@ -123,7 +124,7 @@ endif
 test: UNIT_TESTS = $(shell go list ./...  | grep -v systest)
 
 test: get-libs
-	$(ULIMIT) CGO_LDFLAGS="$(CGO_TEST_LDFLAGS)" TEST_LOG_LEVEL=$(TEST_LOG_LEVEL) go test -timeout 0 -v -p 1 $(UNIT_TESTS)
+	$(ULIMIT) CGO_LDFLAGS="$(CGO_TEST_LDFLAGS)" TEST_LOG_LEVEL=$(TEST_LOG_LEVEL) gotestsum -- -timeout 0 -p 1 $(UNIT_TESTS)
```

Comment:
would `TEST_DIRECTORY` instead of `UNIT_TEST` work here as well? It's part of `gotestsum`'s documentation.

Also: why are there two `test` targets defined in this `Makefile`?

Reply:
My guess is that both `test` target definitions are executed. 

Using `TEST_DIRECTORY` could work, I think. I didn't see any benefit so I limited the scope of changes to a minimum.

Reply:
I would still get rid of having two `test` targets and instead set `UNIT_TESTS` or `TEST_DIRECTORY` in the same block as calling `gotestsum`.